test/server/listener_manager_impl_test.cc

```diff
@@ -791,63 +797,103 @@ version_info: version1
   }
 }
 
-TEST_F(ListenerManagerImplTest, OverrideListener) {
-  time_system_.setSystemTime(std::chrono::milliseconds(1001001001001));
-
+TEST_F(ListenerManagerImplTest, RuntimeDisabledInPlaceUpdateFallbacksToTraditionalUpdate) {
```

Comment:
To make the diffs in here easier to understand, please do not modify any of the existing tests and just add new tests. If it's easier to do just disable the feature in the old tests, I would add a test fixture variant to do that and keep all existing tests the same, then add new in place update tests. This will make it easier to understand and revert if needed. Later, when we remove the guard we can cleanly remove the old tests.